examples/templates/async/example.fabric

```diff
@@ -0,0 +1,61 @@
+document "example" {
+  title = "Document title"
+
+  data sleep data_a {
+    duration = "2s"
+  }
+  data sleep data_b {
+    duration = "1s"
+  }
+
+  data sleep data_c {
+    duration = "1s"
+  }
+
+  data sleep data_d {
+    duration = "3s"
+  }
+
+  content title {
+    value = "A"
+  }
+
+  content sleep content_a {
+    duration = "2s"
+  }
+
+  content title {
+    value = "B"
+  }
+
+  content sleep content_b {
+    duration = "1.5s"
+  }
+
+  content title {
+    value = "C"
+  }
+
+  content sleep content_c {
+    duration = "3.5s"
+    depends_on = ["content_a", "content_b"]
```

Comment:
The names here should be full, I think -- `content.sleep.content_a` and `content.sleep.content_b`, to a) avoid name conflicts b) to make it easier to understand the dependencies